docs/Client.asciidoc

```diff
@@ -111,6 +111,16 @@ curl -u arthur:1234567890ABCDEF:ABCDEF1234567890 -X DELETE \
     https://openqa.example.com/api/v1/assets/1
 ----
 
+And for HTTP clients that don't support Basic authentication or where the use
```

Comment:
Could I suggest `s/And for HTTP clients/Additionally, regarding HTTP clients/`

I just think gives more emphasis to the alternative than a `And...`